### PR TITLE
Migration cleanup

### PR DESCRIPTION
**Pull request by ru-fu**
A small follow-up to the migration docs - move the live migration content from the container page to the new page that covers live migration.

### CODE REVIEW

File: doc/instances.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The currently supported keys are:
 ```{rst-class} dec-font-size break-col-1 min-width-1-15
 ```
 
+(instance-configuration)=
```
Comment (tomponline) [inline doc/instances.md:50]: This bit seems unrelated to the commit message "doc: move live migration content"

**Comment by tomponline** [comment]
@ru-fu I think I missed this on the original live migration PR, but in this section https://linuxcontainers.org/lxd/docs/master/howto/move_instances/#live-migration I think we will need to cover that in order to allow VM live migration to work the VM needs to have support for stateful stop be enabled. This means one would need to set `migration.stateful=true` on the instance and `size.state` to at least the size of the VM's `limits.memory` setting. Otherwise live migration cannot work for VMs at this time.

As we periodically get questions about this on the forum: https://discuss.linuxcontainers.org/t/creating-a-stateful-vm/14951

Do you agree @stgraber ?

File: doc/instances.md (hunk 1 above)
Comment (ru-fu) [reply doc/instances.md:50]: I need the ID to be able to link to the options, which are part of the live migration content.

**Comment by ru-fu** [comment]
> @ru-fu I think I missed this on the original live migration PR, but in this section https://linuxcontainers.org/lxd/docs/master/howto/move_instances/#live-migration I think we will need to cover that in order to allow VM live migration to work the VM needs to have support for stateful stop be enabled. This means one would need to set `migration.stateful=true` on the instance and `size.state` to at least the size of the VM's `limits.memory` setting. Otherwise live migration cannot work for VMs at this time.
> 
> As we periodically get questions about this on the forum: https://discuss.linuxcontainers.org/t/creating-a-stateful-vm/14951
> 
> Do you agree @stgraber ?

See https://github.com/lxc/lxd/pull/10889
